### PR TITLE
SAIC-353 Functional tests for single tenant

### PR DESCRIPTION
**Pull request by ntymtsiv**

### CODE REVIEW

**Comment by ntymtsiv** [comment]
Ready to merge after review.

File: devlab/tests/filtering_utils.py [hunk 1]
```diff
@@ -14,14 +14,14 @@
 
 import os
 import yaml
-
+import config
```
Comment (realsystem) [inline devlab/tests/filtering_utils.py]: Did you check it by flake8 or pep?

Comment (roman-verchikov) [inline devlab/tests/filtering_utils.py]: Yep, looks like there's couple of pep8 violations: 
```
$ flake8  devlab/tests/filtering_utils.py
devlab/tests/filtering_utils.py:19:1: E302 expected 2 blank lines, found 1
devlab/tests/filtering_utils.py:24:80: E501 line too long (81 > 79 characters)
```
Please fix

Comment (ntymtsiv) [inline devlab/tests/filtering_utils.py]: fixed

**Pull request merged**